formats/shared/definitions/hmrc_manual.jsonnet

```diff
@@ -95,4 +95,20 @@
       },
     },
   },
+  hmrc_manual_section_parent: {
+    description: "The parent manual for a manual section",
+    type: "object",
+    additionalProperties: false,
+    required: [
```

Comment:
We should probably require the `title` too as it will be a required field when we actually start using it

Reply:
I explicitly didn't make it required in this step because it causes the search-api and hmrc-manuals-publisher tests to fail, but I've put a checklist step into the card to make it a required property once the other tasks are done (ie when all the existing data actually would actually adhere to the schema with that property required).